sdk/python/feast/errors.py

```diff
@@ -85,10 +85,11 @@ def __init__(self, online_store_class_name: str):
         )
 
 
-class FeastOnlineStoreConfigInvalidName(Exception):
-    def __init__(self, online_store_config_class_name: str):
+class FeastStoreConfigInvalidName(Exception):
+    def __init__(self, online_store_config_class_name: str, store_type="Online"):
```

Comment:
Is there a reason to set a default here?

Reply:
Nope, going to clean this up

Reply:
Also, should `online_store_config_class_name` be online store agnostic?

Reply:
done